code/game/objects/items/stacks/tiles/tile_types.dm

```diff
@@ -76,6 +76,17 @@
 	turf_type = /turf/open/floor/grass
 	resistance_flags = FLAMMABLE
 
+/obj/item/stack/tile/grass/attackby(obj/item/W, mob/user, params)
+	if((W.tool_behaviour == TOOL_SHOVEL) && params)
+		to_chat(user, "<span class='notice'>You start digging up [src]</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You start digging up [src].</span>")
```

Reply:
Again, a direct copy of grass turf behavior.